packages/flutter_driver/lib/src/common/wait.dart

```diff
@@ -0,0 +1,324 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/scheduler.dart';
```

Comment:
Since this is in common, it is shared by both the application and the driver script. The latter doesn't have access to `dart:ui`, which is why this fails to run.

Reply:
Thank you Jonah!

I'm trying to run the devicelab tests locally. Do you happen to have a result link to the failing devicelab tests?

Reply:
Nm. I was able to get one test failure due to `dart:ui`. Looking.